host/greatfet/commands/greatfet_firmware.py

```diff
@@ -155,20 +154,8 @@ def main():
                         help="Read data into file", default='')
     parser.add_argument('-w', '--write', dest='write', metavar='<filename>', type=str,
                         help="Write data from file", default='')
-    parser.add_argument('-s', '--serial', dest='serial', metavar='<serialnumber>', type=str,
-                        help="Serial number of device, if multiple devices", default=None)
-    parser.add_argument('-i', '--index', dest='index', metavar='<i>', type=int,
-                        help="number of the attached device (default: 0)", default=0)
-    parser.add_argument('-q', '--quiet', dest='quiet', action='store_true',
```

Comment:
It used to be that `greatfet_firmware` output nothing by default.  As a new user, I found this confusing and changed it (see #43).  I'm not able to run `greatfet_firmware` to see what it does now, but if it's back to showing nothing, considering showing output by default.